docs/src/main/asciidoc/security-openid-connect-client-reference.adoc

```diff
@@ -3,7 +3,7 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
-= OpenID Connect (OIDC) and OAuth2 Client and Filters Reference Guide 
+= OpenID Connect (OIDC) and OAuth2 Client and Filters Reference Guide
```

Comment:
Not sure if you are focussing on titles in this iteration of the structure, but if so:
- Sentence style capitalization
- Avoid using 'reference' in a reference topic title

See "[Quarkus docs style guide](https://github.com/quarkusio/quarkus/blob/main/docs/src/main/asciidoc/doc-reference.adoc)"

Reply:
I want to limit the focus of this MR solely to restructuring. This line pops up as a change because a previous contributor didn't have their editor configured to trim trailing white spaces, which might be a Red Hat standard.

Reply:
Thanks for the reference. I've bookmarked it and will review it closely. At some point, we should perform style reviews for ups-downs content.

Reply:
This MR is focused purely on restructuring. I'd like to handle style issues separately.

Reply:
I'm revisiting this question. I think the correct answer is that we should refactor titles only for topics we are recomposing and their offspring. For example, in this case, `security-oidc-bearer-authentication-concept.adoc` and each of the new files. We probably shouldn't update titles (and by extension filenames) in secondary files, which are affected by the recomposition of the primary file. These secondary files should mostly contain updated xrefs.